hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BytesBuffer.java

```diff
@@ -318,6 +326,7 @@ public BytesBuffer writeId(Id id, boolean big) {
                 len -= 1;
                 int high = len >> 8;
                 int low = len & 0xff;
+                assert high != 0x7f; // The tail 7 bits 1 reserved for UUID
```

Comment:
'len <= BIG_ID_MAX_LEN' can't ensure 'high != 0x7f',

Reply:
id len must be in range [0, 126]